### DISCUSSION TITLE
Here's a little shield thingy.

### DISCUSSION THREAD

**Discussion by ramphands** [probonopd/MiniDexed#371] [General]
https://github.com/ramphands/Minidexed-Shield

Hope it's of use to some of you.
Codec board attaches underneath.
The Pi Pico functions as a bridge for connecting two USB hosts together - I use a Monome Norns to sequence the Minidexed.  You can omit this if you wish, but it adds some nice functionality if you want to interface with a PC for example.  
![0](https://user-images.githubusercontent.com/44291906/197391318-0c0f3f19-17ba-4baa-8f30-b811ec101883.jpg)

**Comment by diyelectromusic** [comment]
That looks great!  Great idea adding the Pico too :)
